src/app/review/Review.js

```diff
@@ -81,6 +81,7 @@ class Review extends Component {
     const { params } = this.props.match;
     this.fetchDeck(params.deckId);
     window.addEventListener("keyup", e => this.onKeyPress(e));
+	window.addEventListener("keydown", e => this.onKeyRelease(e));
```

Comment:
When the `Review` component is unmounted, `window` should no longer be listening for this event. Can you remove this listener inside `componentWillUnmount`?